lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -127,6 +127,7 @@
     from azure.mgmt.dns import DnsManagementClient
     from azure.mgmt.web import WebSiteManagementClient
     from azure.mgmt.containerservice import ContainerServiceClient
+    from azure.mgmt.keyvault import KeyVaultManagementClient
```

Comment:
no need to create KeyVaultManagementClient in common module, better to create it locally in keyvault module